site/bulletin_board/routes/users.js

```diff
@@ -84,4 +86,26 @@ router.post('/', (req, res, next) => {
   });
 });
 
+router.get('/profile', (req, res, next) => {
+  res.redirect("profile/"+req.session.passport.user);
```

Comment:
Minor note; one of the benefits of middleware like passport is that you can swap it out for something else. But here you are accessing `req.session.passport` which tightly couples your code to Passport. Since Passport populates `req.user` (which is "standardized") it would probably be better to use `req.user` consistently.